galaxy_ng/app/urls.py

```diff
@@ -47,6 +47,14 @@
     path("healthz", views.health_view),
 ]
 
+if settings.get("API_ROOT") != "/pulp/":
```

Comment:
do we need this conditional since we made this not user-configurable?

Reply:
I like to cover the bases, but probably not